views/package_version/show.ejs

```diff
@@ -1,6 +1,7 @@
 <%- partial ('../shared/_navbar.ejs') %>
 <% var pkgVersion = data; %>
 <section class="package" itemscope itemtype="http://schema.org/SoftwareApplication">
+<section data-package-name="<%=pkgVersion.package_name%>" data-type-id="<%=pkgVersion.package['type_id']%>" class="packageData" itemscope itemtype="http://schema.org/SoftwareApplication"></section>
```

Comment:
What I meant is put the `data-*` in the section on line 3, no need to create a new empty section, you can add your class and the other data-* to the existing one ;)